Orange/data/io.py

```diff
@@ -749,6 +749,10 @@ class CSVReader(FileFormat):
     SUPPORT_SPARSE_DATA = False
     PRIORITY = 20
 
+    def __init__(self, filename):
+        super().__init__(filename)
+        csv.field_size_limit(100*1024*1024)
```

Comment:
This is a global side-effect. Put the line on top in a global namespace, somewhere shorty below the `csv` import. Precede by a comment:
> Support values longer than 128K (i.e. text contents features)